Formula/jthread.rb

```diff
@@ -0,0 +1,34 @@
+class Jthread < Formula
+  desc "C++ class to make use of threads easy"
+  homepage "https://research.edm.uhasselt.be/jori/jthread"
+  url "https://research.edm.uhasselt.be/jori/jthread/jthread-1.3.3.tar.bz2"
+  sha256 "17560e8f63fa4df11c3712a304ded85870227b2710a2f39692133d354ea0b64f"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.cpp").write <<~EOS
+      #include <jthread/jthread.h>
+      using namespace jthread;
+      int main() {
+        JMutex* jm = new JMutex();
+        jm->Init();
+        jm->Lock();
+        jm->Unlock();
+        return 0;
+      }
+    EOS
+
+    libs = `pkg-config --libs jthread`.chomp
+    cflags = `pkg-config --cflags jthread`.chomp
+
+    system "#{ENV.cxx} test.cpp -o test #{libs} #{cflags}"
```

Comment:
Once you've removed the temporary variables above, just separate the arguments here. See how `libogg` is doing it.